build.gradle

```diff
@@ -99,7 +99,7 @@ ext.deps = [
         math3               : 'org.apache.commons:commons-math3:3.0',
         metricsCore         : 'io.dropwizard.metrics:metrics-core:3.1.0',
         metricsJvm          : 'io.dropwizard.metrics:metrics-jvm:3.1.0',
-        mockito             : 'org.mockito:mockito-all:1.10.19',
+        mockito             : 'org.mockito:mockito-core:1.10.19',
```

Comment:
Is it done so that mockito doesn't bring its own version of Hamcrest which causes conflict with the intellij's builtin version?

Reply:
Found this https://solidsoft.wordpress.com/2012/09/11/beyond-the-mockito-refcard-part-3-mockito-core-vs-mockito-all-in-mavengradle-based-projects/

Could you add a comment here so that people don't accidentally change back to -all?

Reply:
Also https://github.com/mockito/mockito/issues/324
"mockito-all is deprecated and is no longer published starting with version 2.0.3-beta to not confuse users."

Looks like we can update the mockito itself. The latest is
https://mvnrepository.com/artifact/org.mockito/mockito-core/2.10.0
If you update the version, then no comment is necessary since the -all version is no longer published.